MapleServer2/Commands/Game/CurrencyCommands.cs

```diff
@@ -0,0 +1,49 @@
+﻿using MapleServer2.Commands.Core;
+
+namespace MapleServer2.Commands.Game
+{
+    public class CurrencyCommands : InGameCommand
+    {
+        public CurrencyCommands()
+        {
+            Aliases = new[]
+            {
+                "currency"
+            };
+            Description = "Set the amount of the currency type specified.";
+            AddParameter<string>("name", "Type of the currency you want to add.");
+            AddParameter<long>("amount", "Amount of the currency type.");
+        }
+
+        public override void Execute(GameCommandTrigger trigger)
+        {
+            string currencyName = trigger.Get<string>("name");
+            long amount = trigger.Get<long>("amount");
+
```

Comment:
Add an `if currencyName empty or null` the possible currencies that can be used and if amount < 0, return with or without an message